market/demos/excel-connector/meta.json

```diff
@@ -0,0 +1,73 @@
+{
+	"id": "excel-connector-demo",
```

Comment:
I'd remove the "-demo" postfix. Actually it's not just a 'demo' we're sharing here. But a real excel-connector + plus a demo.

Reply:
Is the location https://github.com/axonivy/market/tree/master/market/demos correct?
or is it better to place the connector at https://github.com/axonivy/market/tree/master/market/connector?

Reply:
ahh good catch: the location should also be changed -> market/connector/exel-connector